app/src/main/kotlin/com/wire/android/ui/calling/ongoing/participantsview/ParticipantTile.kt

```diff
@@ -286,6 +288,7 @@ private fun UsernameTile(
     hasEstablishedAudio: Boolean
 ) {
     val color = if (isSpeaking) MaterialTheme.wireColorScheme.primary else Color.Black
+    val nameLabelColor = if (hasEstablishedAudio) Color.White else colorsScheme().secondaryText
```

Comment:
thought: maybe we should move all these hard-coded to the theme colors as well, they would just be the same for both light and dark mode, but it would ensure we have all colors in one place so if we decide to change one of them we wouldn't need to look for them in the whole project

Reply:
Well, not that simple by just changing the color in the file.. we need to check the whole project in all cases I would say especially for white and black